pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/NonPersistentTopics.java

```diff
@@ -132,7 +132,7 @@ public void getInternalStats(
                 })
                 .thenAccept(asyncResponse::resume)
                 .exceptionally(ex -> {
-                    if (!isRedirectException(ex)) {
+                    if (!isNot307And404Exception(ex)) {
```

Comment:
Looks like this line and all the others where `!isRedirectException(ex)` has been replaced with `!isNot307..` are wrong, as the new line should NOT have the negation exclamation mark any more.